xml/admin_ceph_troubleshooting.xml

```diff
@@ -518,4 +518,35 @@ keep_jobs: 1</screen>
    </procedure>
   </tip>
  </sect1>
+ <sect1 xml:id="trouble-osd-panic">
+  <title>OSD Panic Occurs when Media Error Happens during &filestore; Directory Split</title>
+
+  <para>
+   When directory split error is generated on &filestore; OSDs, the
+   corresponding OSD is designed to terminate. It is then easier to detect the
+   problem and avoids introducing inconsistencies. If the OSD terminates
+   frequently, &systemd; may disable it permanently depending on the &systemd;
+   configuration. After the OSD is disabled, it will be set out and process of
+   data migration will be started.
+  </para>
+
+  <para>
+   You should notice the OSD termination by regularly running the <command>ceph
+   status</command> command and perform necessary steps to investigate the
```

Comment:
_the_ necessary steps

(if it is unambiguous what the steps are)

or

_any_ necessary steps

(if there is a choice of possible steps)